### ISSUE TITLE
Wagtail userbar toggle doesn't close the userbar

### ISSUE THREAD

**Issue by Lovelyfin00** [wagtail/wagtail#9342]
### Issue Summary
When the user bar toggle button is toggled, the the items display but when it's toggled again and the items is supposed to disappear, they don't
 So when the user clicks on the user bar toggle button, the content displays fine. But when it's clicked again and the content is supposed to disappear, they don't. The content  disappears for some seconds and then reappears again. Then also, the first list item of has a yellow border around it when the toggle event is triggered.

### Steps to Reproduce

1. Start the wagtail bakery
2. Navigate to the fixed user bar nav on the bottom right corner
3. Toggle the button

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

-   I expected that the second I click the button, the content is supposed to disappear or display none.

-   I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: (yes / no) yes

![user bar image 2](https://user-images.githubusercontent.com/38161296/195600930-af1eee77-e8be-43b0-9654-5842eabc3eaf.png)

@thibaudcolas  @lb-  Please confirm. Thanks!

**Comment by vicky1312** [comment]
Can I work on this? @Lovelyfin00 @thibaudcolas

**Comment by Lovelyfin00** [comment]
I'm going to work on it after the mentors has confirmed it

**Comment by vicky1312** [comment]
Alright then

**Comment by AnishaSingh0118** [comment]
Is this issue still not confirmed?

**Comment by Lovelyfin00** [comment]
No it hasn't

**Comment by AnishaSingh0118** [comment]
> No it hasn't

Oh! Ok thanks for confirming.

**Comment by thibaudcolas** [comment]
Confirmed! Thank you for the report @Lovelyfin00.

I’m not sure what’s happening here, but yes you’d expect clicking the toggle to _close_ the userbar when it’s opened.

Relevant code: https://github.com/wagtail/wagtail/blob/main/client/src/entrypoints/admin/userbar.js

**Comment by AnishaSingh0118** [comment]
Hello @thibaudcolas as I had an idea to resolve this issue or to reach to the point where it gets resolved, I tried to make efforts and have come up with a PR. 

Kindly guide what else could be done. 
Would appreciate your feedback!

**Comment by Lovelyfin00** [comment]
Hi @AnishaSingh0118
I was going to work on it
But since you have already done that, then it's cool
Before working or creating a PR for other issues, find out if someone else is already working on it

**Comment by AnishaSingh0118** [comment]
> Hi @AnishaSingh0118 I was going to work on it But since you have already done that, then it's cool Before working or creating a PR for other issues, find out if someone else is already working on it

I have not entirely resolved the issue, you could still do it.

**Comment by lb-** [comment]
hey all - I think you should keep at this @AnishaSingh0118 

@Lovelyfin00  - I am sure there are many other things to get your help on, and you are actively already working on some at the moment in parallel.

I think there has been a bit of confusion on this issue as it was not confirmed as something we wanted to fix until recently.

Thanks all for wanting to help improve Wagtail but Anisha - I encourage you to read the feedback on your PR, apply those suggested changes and see how that works for you.

**Comment by AnishaSingh0118** [comment]
> hey all - I think you should keep at this @AnishaSingh0118
> 
> @Lovelyfin00 - I am sure there are many other things to get your help on, and you are actively already working on some at the moment in parallel.
> 
> I think there has been a bit of confusion on this issue as it was not confirmed as something we wanted to fix until recently.
> 
> Thanks all for wanting to help improve Wagtail but Anisha - I encourage you to read the feedback on your PR, apply those suggested changes and see how that works for you.

Yes, I am working on it. Thank you so much for your support and for clearing this confusion.

**Comment by Lovelyfin00** [comment]
Yeah @lb-
It was confirmed late
Thanks for working on this @AnishaSingh0118
